src/tss2-fapi/api/Fapi_NvRead.c

```diff
@@ -261,7 +261,7 @@ Fapi_NvRead_Finish(
         command->nv_obj = object->misc.nv;
 
         if (size)
```

Comment:
Do we still need the check here then?

Reply:
Thanks for the hint. I have removed the check.